packages/katello/rubygem-foreman_scc_manager/rubygem-foreman_scc_manager.spec

```diff
@@ -1,23 +1,16 @@
 # template: foreman_plugin
-%{?scl:%scl_package rubygem-%{gem_name}}
-%{!?scl:%global pkg_name %{name}}
-
 %global gem_name foreman_scc_manager
 %global plugin_name scc_manager
-%global foreman_min_version 3.4
-%global katello_min_version 3.16.0
+%global foreman_min_version 2.1
```

Comment:
This comes from https://github.com/ATIX-AG/foreman_scc_manager/blob/a84d26d6aeb11b03b52c290c515b64ef6794066b/lib/foreman_scc_manager/engine.rb#L25
The old RPM spec said 3.4, who is right? I don't know :)

Reply:
I think the Katello >= 3.16 version implies a much newer version than 2.1 anyway.

Reply:
Nope, that's exactly the one…

https://github.com/theforeman/forklift/blob/c60a2d17d9dbdb3a33c7190a4beb0f1ab06ed6e4/vagrant/config/versions.yaml#L22-L31

Reply:
3.4 should be correct, also taking into account the configs in package.json file. We should update the gemspec, eventually.

Reply:
Then please update `engine.rb` (and the gemspec) and on next run it will regenerate that line correctly. :)

Reply:
(given 3.4 is old enough, I see no problem in merging as is, we won't be backporting that regeneration anyway, and if we decide to, not to older than 3.5 ;))